### DISCUSSION TITLE
Check options SQLServer

### DISCUSSION THREAD

**Discussion by pablim** [dbeaver/dbeaver#624] [Q&A] [answered]
I can't check the options for create a new column for a table in SQLServer.

![criar colunas dbeaver](https://cloud.githubusercontent.com/assets/11844740/17102174/cc439bda-524e-11e6-9604-e14c518782ea.png)

**Comment by pablim** [comment]
More specifically Auto Increment, Key and Description

**Comment by serge-rider** [comment, marked as answer]
These options are not supported for SQL Server (yet). 
SQL Server driver uses Generic plugin which means plain JDBC and no database specific options.

We are working on this new plugin..

**Comment by pablim** [comment]
Ok! Thanks for feedback.
